### DISCUSSION TITLE
Upvote `aria-label` is identical across comments

### DISCUSSION THREAD

**Discussion by ColeFrench** [community/community#65389] [Discussions]
### Select Topic Area

Bug

### Body

The upvote button for the comment you're reading right now has an `aria-label` of "Upvote: (1): ColeFrench, 06:43PM on August 29".[^1] This makes sense, because the user should know how many upvotes there are and whose comment they are upvoting.

However, if you leave a comment on this discussion, your upvote button will have the same `aria-label`, yet I would expect it to identify *you* as the user with the date and time *you* commented.

[^1]: In all likelihood, the date/time depends on your local timezone.

**Comment by dakshithadissanayaka** [comment]
You're correct that the aria-label for the upvote button should change for the user's own comments to provide more context and accessibility. The current behavior is inconsistent and could lead to confusion for users, especially those who rely on screen readers or other assistive technologies.

To improve the user experience and accessibility, the aria-label for the upvote button should dynamically update based on the user and the comment being upvoted. For the user's own comments, the aria-label should include their username and the timestamp of their comment, similar to how it currently displays for other users' comments.

Here's an example of how the aria-label could be updated for the user's own comments:

```html
Upvote: You, 06:43PM on August 29
```

This would provide more relevant information to the user and improve the overall accessibility of the discussions feature.
